index.js

```diff
@@ -39,10 +39,14 @@ async function terraform(args) {
 
 (async () => {
   const terraformDirectory = core.getInput('terraform_directory');
-  let terraformDoApply = core.getInput('terraform_do_apply') === 'true';
-  let terraformDoDestroy = core.getInput('terraform_do_destroy') === 'true';
-  const terraformLock = core.getInput('terraform_lock');
+  let terraformDoApply = core.getBooleanInput('terraform_do_apply');
+  let terraformDoDestroy = core.getBooleanInput('terraform_do_destroy');
+  const terraformLock = core.getBooleanInput('terraform_lock');
   const terraformParallelism = core.getInput('terraform_parallelism');
+  const terraformTargets = core.getInput('terraform_targets')
+    .split(',')
+    .filter(_str => _str.trim() !== '')
+    .map(_str => `-target '${_str.trim()}'`);
```

Comment:
Inspired by `core.getMultilineInput` (actions/toolkit#829). Open to using that instead, though I would have preferred support for actual lists.

Reply:
`core.getMultilineInput` feels like a nice solution at first sight.

Reply:
It's nice, but splits only by newlines, so that would result in either of these:
```
terraform_targets: 'foo\nbar\nbaz'
```

```
tarraform_targets: |
  foo
  bar
  baz
```

In case of the latter, I'm not even sure how it parses the leading spaces, as the library doesn't strip.

Reply:
I'd go for the latter as that looks the cleanest imho. As for whitespace handling, that's part of the yaml spec 😉